From 15616ffd51dba116095bd1c395d377df9a5bc0f4 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 11 May 2022 15:40:02 +0100
Subject: [PATCH] overlays: Remove other leading zeroes from node addresses

Newer versions of dtc warn about node addresses with leading zeroes.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts | 2 +-
 arch/arm/boot/dts/overlays/imx219-overlay.dts      | 2 +-
 arch/arm/boot/dts/overlays/imx258-overlay.dts      | 2 +-
 arch/arm/boot/dts/overlays/ov5647-overlay.dts      | 2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts b/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
index 743f14ae5768..1ab71e653e20 100644
--- a/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
+++ b/arch/arm/boot/dts/overlays/fe-pi-audio-overlay.dts
@@ -37,7 +37,7 @@ __overlay__ {
 			#size-cells = <0>;
 			status = "okay";
 
-			sgtl5000@0a {
+			sgtl5000@a {
 				#sound-dai-cells = <0>;
 				compatible = "fsl,sgtl5000";
 				reg = <0x0a>;
diff --git a/arch/arm/boot/dts/overlays/imx219-overlay.dts b/arch/arm/boot/dts/overlays/imx219-overlay.dts
index d73f76aba8b6..0134fb2677c6 100644
--- a/arch/arm/boot/dts/overlays/imx219-overlay.dts
+++ b/arch/arm/boot/dts/overlays/imx219-overlay.dts
@@ -64,7 +64,7 @@ imx219_0: endpoint {
 				};
 			};
 
-			vcm: ad5398@0c {
+			vcm: ad5398@c {
 				compatible = "adi,ad5398";
 				reg = <0x0c>;
 				status = "disabled";
diff --git a/arch/arm/boot/dts/overlays/imx258-overlay.dts b/arch/arm/boot/dts/overlays/imx258-overlay.dts
index 12326040c457..7a8ed540438e 100644
--- a/arch/arm/boot/dts/overlays/imx258-overlay.dts
+++ b/arch/arm/boot/dts/overlays/imx258-overlay.dts
@@ -74,7 +74,7 @@ imx258_0: endpoint {
 				};
 			};
 
-			vcm: ad5398@0c {
+			vcm: ad5398@c {
 				compatible = "adi,ad5398";
 				reg = <0x0c>;
 				status = "disabled";
diff --git a/arch/arm/boot/dts/overlays/ov5647-overlay.dts b/arch/arm/boot/dts/overlays/ov5647-overlay.dts
index af2c86383e1f..d06ee9229b0d 100644
--- a/arch/arm/boot/dts/overlays/ov5647-overlay.dts
+++ b/arch/arm/boot/dts/overlays/ov5647-overlay.dts
@@ -39,7 +39,7 @@ ov5647_0: endpoint {
 				};
 			};
 
-			vcm: ad5398@0c {
+			vcm: ad5398@c {
 				compatible = "adi,ad5398";
 				reg = <0x0c>;
 				status = "disabled";
-- 
2.32.0

